static/js/components/analytics/AnalyticsPane.js

```diff
@@ -0,0 +1,177 @@
+        {this.renderHeader()}
+        {this.renderBody()}
+      </section>
+    )
+  }
+
+  renderHeader() {
+    const { video } = this.props
+    const videoUrl = `${window.location.origin}${makeVideoUrl(video.key)}`
+    return (
+      <header className={`${CSS_NS}-header`}>
+        <div
+          style={{
+            display:        "flex",
+            flexDirection:  "row",
+            justifyContent: "space-between"
+          }}
+        >
+          <h2
+            className={`${CSS_NS}-video-title`}
+            style={{
+              margin:     0,
+              fontWeight: "500",
+              fontSize:   "large"
+            }}
+          >{`Views per minute for video "${video.title}"`}</h2>
+        </div>
+        <a
+          className={`${CSS_NS}-video-link`}
+          style={{ fontSize: "small" }}
+          href={videoUrl}
+        >
+          {videoUrl}
+        </a>
+      </header>
+    )
+  }
+
+  renderBody() {
+    const { analyticsData } = this.props
+    const bodyContent = this.analyticsDataIsEmpty(analyticsData)
+      ? this.renderEmptyDataMessage()
+      : this.renderTabs()
+    return (
+      <section className={`${CSS_NS}-body`} style={{ width: "100%" }}>
+        {bodyContent}
+      </section>
+    )
+  }
+
+  analyticsDataIsEmpty(analyticsData: any): boolean {
+    return !!analyticsData && _.isEmpty(analyticsData.times)
+  }
+
+  renderTabs() {
+    return (
+      <section>
+        <TabBar
+          activeTabIndex={this.state.activeTabIndex}
+          onChange={evt => this.setState({ activeTabIndex: evt.target.value })}
+          style={{
+            marginLeft:   0,
+            borderBottom: "thin solid #333",
+            width:        "100%"
+          }}
+        >
+          {this._tabDefs.map(tabDef => {
+            return <Tab key={tabDef.title}>{tabDef.title}</Tab>
+          })}
+        </TabBar>
+        <section>{this._tabDefs[this.state.activeTabIndex].renderFn()}</section>
+      </section>
+    )
+  }
+
+  renderEmptyDataMessage() {
+    return (
+      <section
+        className={`${CSS_NS}-empty-data-message`}
+        style={{
+          textAlign:  "center",
+          marginTop:  "1em",
+          paddingTop: "1em",
+          borderTop:  "thin solid grey"
+        }}
+      >
+        <div>No analytics data for this video at this time.</div>
+      </section>
+    )
+  }
+
+  renderChart() {
+    return (
+      <AnalyticsChart
+        analyticsData={this.props.analyticsData}
+        channelLabelFn={this.generateLabelForChannel}
+        style={{ width: "100%" }}
+      />
+    )
+  }
+
+  renderTable() {
+    return (
+      <AnalyticsTable
+        analyticsData={this.props.analyticsData}
+        channelLabelFn={this.generateLabelForChannel}
+        style={{ width: "100%" }}
+      />
+    )
+  }
+
+  generateLabelForChannel(channel: string) {
+    // Hack to make zero-based camera names more suitable for humans.
+    if (!channel) {
```

Comment:
On closer look, I think the real analytics data is already labeled `camera1` through `camera4`, and it's just the mock data that is 0-3?

Reply:
@mbertrand ah, you're right, the cameras are 1-4:
https://github.com/mitodl/odl-video-service/pull/519/files#diff-3ba30cee865fa843dc5141d396244367L37

I'll change the mock data and take out the extra label formatting.